xml/System/Type.xml

```diff
@@ -10757,9 +10757,16 @@ GetType(Array).IsAssignableFrom(type)
         <ReturnType>System.Boolean</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a value indicating whether the <see cref="T:System.Type" /> is a byref-like structure.</summary>
+        <value><see langword="true" /> if the <see cref="T:System.Type" /> is a a byref-like structure; otherwise, <see langword="false" />.</value>
```

Comment:
```suggestion
        <value>
          <see langword="true" /> if the type is a a byref-like structure; otherwise, <see langword="false" />.</value>
```